src/harness/fourslash.ts

```diff
@@ -841,7 +841,16 @@ namespace FourSlash {
             assert.equal(actual.hasAction, hasAction);
             assert.equal(actual.isRecommended, isRecommended);
             assert.equal(actual.source, source);
-            assert.equal(actual.sortText, sortText || ts.Completions.SortText.LocationPriority, this.messageAtLastKnownMarker(`Actual entry: ${JSON.stringify(actual)}`));
+
+            let isSortTextEqual: boolean;
+            if (!sortText) {

```

Comment:
No. We want the default always LocationPriority. Please go ahead and update all tests that fail because of this instead.